documentation/SA1202.md

```diff
@@ -40,6 +40,12 @@ rule.
 Complying with a standard ordering scheme based on access level can increase the readability and maintainability of the
 file and make it easier to identify the public interface that is being exposed from a class.
 
+### Interfaces and default interface members
+
+Default interface members (including members with implementations and static interface members) follow the same access
+ordering as class members. Within an interface, `public` members should appear before `internal`, `protected internal`,
+`protected`, `private protected`, and `private` members.
```

Comment:
Interface member ordering is _not_ mere style, but is significant when the interface is used as a COM interface. 
Should accessibility modifiers and member type sorting rules automatically be disabled when an interface is annotated with attributes typically associated with COM?